src/lib.rs

```diff
@@ -1,23 +1,27 @@
-use tracing::{
-    subscriber::{set_global_default, SetGlobalDefaultError}, Level,
+use tracing::subscriber::{set_global_default, SetGlobalDefaultError};
+
+pub use tracing::{
+    debug, debug_span, error, error_span, info, info_span, span, trace, trace_span, warn,
+    warn_span, Level,
```

Comment:
I think reexporting the whole crate under it's own name is better than exporting a group of types right in dioxus-logger itself, right?

```suggestion
pub use tracing;

use tracing::{
    debug, debug_span, error, error_span, info, info_span, span, trace, trace_span, warn,
    warn_span, Level,
```

Reply:
It's just an idea anyway! Feel free to go whatever route you want